### PR TITLE
Add event sections and speaker data

### PR DESCRIPTION
**Pull request by Siilwyn**
It looks :hankey: , but it works. Adds new data and some initial styling, and it unbreaks the CI ;)

### CODE REVIEW

File: pages/_language/events/_event/index.vue [hunk 1]
```diff
@@ -1,41 +1,100 @@
...
+            >
+              <nuxt-link
+                class="language-switch__link"
+                :class="{ 'language-switch__link--active': params.language === locale }"
+                rel="alternate"
+                :hreflang="locale"
+                :lang="locale"
+                :to="`/${locale}/events/${params.event}`"
+              >
+                {{ locale }}
```
Comment (BasVoorhoede) [inline pages/_language/events/_event/index.vue:23]: The locale is presented here as a language code, but is it maybe possible to display it in the language of the user? If not, the `lang`-attribute is not necessary, as it is in the same language as the page language.

File: pages/_language/events/_event/index.vue [hunk 2]
```diff
@@ -1,41 +1,100 @@
...
+          </span>
+          <span class="event-header__title">
+            {{ event.location }} {{ event.name }}
+          </span>
+        </h2>
+      </div>
+    </header>
+    <main class="event__body">
+      <aside class="wwa-mention">
+        <img class="wwa-mention__image" src="/favicon-96x96.png" width="32" height="32">
```
Comment (BasVoorhoede) [inline pages/_language/events/_event/index.vue:44]: This element is missing an `alt`-tag.

File: pages/_language/events/_event/index.vue (hunk 1 above)
Comment (Siilwyn) [reply pages/_language/events/_event/index.vue:23]: Yes definitely, will need to look into a language mapping perhaps using one of these:
    https://www.npmjs.com/package/locale-code
    https://www.npmjs.com/package/country-locale-map
    https://www.npmjs.com/package/iso-639-1

Think this task is big enough for a new ticket.

File: pages/_language/events/_event/index.vue (hunk 2 above)
Comment (Siilwyn) [reply pages/_language/events/_event/index.vue]: Good catch, since it's just visual I added an empty `alt`!

**Pull request merged**